### ISSUE TITLE
LOMBOK_JAR in eclipserc has a bad path

### ISSUE THREAD

**Issue by tim-tx** [gentoo/eclipse-overlay#21]
If you enable the lombok use-flag for eclipse-sdk-bin-4.6.2, then you get the lombok-bin package and an `eclipse-sdk-bin-4.6.2-eclipserc-bin` file in /etc. This file has a shell variable, `LOMBOK_JAR=/usr/share/lombok/lombok.jar` which gets used in the eclipse launch script. However, the lombok-bin package actually puts the jar into `/usr/share/lombok/lib/lombok.jar` (note `lib`). As it is, lombok is not loaded. If you change the eclipserc file to point to the correct jar location everything works.

Basically, c.f. lines [here](https://github.com/gentoo/eclipse-overlay/blob/77bf4bf0b10577caf3741bc60095c8868962c51a/dev-util/eclipse-sdk-bin/files/eclipse-sdk-bin-4.6.2-eclipserc-bin#L29) and [here](https://github.com/gentoo/eclipse-overlay/blob/77bf4bf0b10577caf3741bc60095c8868962c51a/dev-java/lombok-bin/lombok-bin-1.16.10.ebuild#L29).

**Comment by hartwork** [comment]
Should be fixed now, thanks for the report!
